Credits.md

```diff
@@ -0,0 +1,9 @@
+# Credits
+
+This project is made possible by the people and organizations listed in this document.
+
+## Contributors
+
+<img src="docs/images/id_pass_logo.svg" width="150" height="50">
+
+<img src="docs/images/newlogic_logo.svg" width="100" height="200">
```

Comment:
Could we add the name and a link under the logo like:
```markdown
[Newlogic](https://www.newlogic.com/)
```

Reply:
Have added the links